application/common/helpers/Utils.php

```diff
@@ -206,7 +206,7 @@ public static function getFrontendConfig()
 
         $config = [];
 
-        $config['idpName'] = $params['idpName'];
+        $config['idpName'] = $params['idpDisplayName'];
```

Comment:
Would it be helpful to change `$config['idpName']` to `$config['idpDisplayName']`, as well (since we're still using `idpName` for the IDP's code name)?